src/index.js

```diff
@@ -9,14 +9,12 @@ import App from './App';
 import './index.css';
 import reducer from './reducers';
 
-const store = createStore(
-  reducer,
-  compose(
-    applyMiddleware(thunk),
-    window.__REDUX_DEVTOOLS_EXTENSION__ &&
-      window.__REDUX_DEVTOOLS_EXTENSION__(),
-  ),
-);
+const preloadedState = window.__REDUX_DEVTOOLS_EXTENSION__ &&
```

Comment:
It says `preloadedState` on the version I look at. Maybe I'm missing something, but here it is:
https://github.com/reactjs/redux/blob/master/docs/api/createStore.md